src/context/WagmiProvider/WagmiProvider.tsx

```diff
@@ -0,0 +1,46 @@
+import { getConfig } from 'config'
+import last from 'lodash/last'
+import type { Chain } from 'wagmi'
+import { chain, configureChains, createClient, WagmiConfig } from 'wagmi'
+import { infuraProvider } from 'wagmi/providers/infura'
+import { jsonRpcProvider } from 'wagmi/providers/jsonRpc'
+
+const avalancheChain: Chain = {
```

Comment:
Find a better place and/or make programmatic? See

https://github.com/shapeshift/web/pull/2348/files#diff-c2977d04ff57b046d9667291f82e582192401a3d048c937cc77bee3bc1128440R79-R88

Reply:
Probably the best to leave it here as a const for now, programmatic params getting would be possible but require more work that would better live in its own PR